bazel-build.el

```diff
@@ -24,6 +24,17 @@
 ;;
 ;;; Code:
 
+(defgroup bazel-build nil
+  "Package for building and running code using Bazel."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-build-bazel-program "bazel"
+  "Name of the 'bazel' program for 'bazel-build' and 'bazel-test'."
+  :type 'string
+  :group 'bazel-build
+  :link '(url-link "https://bazel.build/"))
+
 (defun bazel-build (target)
   "Build a Bazel TARGET."
   (interactive (list (bazel-build--read-target "bazel build ")))
```

Comment:
It would be good to also have the prompts here use the the variable instead of hard-coding "bazel".